docs/sync_code.md

```diff
@@ -38,3 +38,41 @@ ticket, edit the description to include the ticket link.
 
 4. **Merge sync PR:**
 After the sync PR has passed github checks and is reviewed and approved, CI will merge it into `rhoai` branch.
+
+## Merging Changes Downstream
+
+```mermaid
+sequenceDiagram
+    actor engineer
+    participant main as opendatahub-io/main
+    participant rhoai as opendatahub-io/rhoai
+    participant rhds as red-hat-data-services/main
+    participant rhoaixy as red-hat-data-services/rhoai-x.y
+
+    engineer ->> main: pull request
+    engineer ->> rhoai: cherry-pick & pull request
+    rhoai ->> rhds: merge (via CI)
+alt landing a change in the next rhoai release
+    rhds ->> rhoaixy: merge (via CI)
+else hotfixing a change into a rhoai release
```

Comment:
do we have definition of "hotfixing" VS just "a change"? 
by looking at diagram,  "hotfix" can be both  for  a z-stream fix and a blocker fix for ongoing release.

Reply:
:+1: I updated the language to call out both explicitly. Please take another look when you have a chance and let me know what you think.